.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,47 +6,71 @@ labels: ''
 assignees: ''
 ---
 
-<!-- IMPORTANT
-Bug reports that do not make an effort to help the developers will be closed without notice.
-Make sure that this bug has not already been opened and/or closed by searching the issues on GitHub, as duplicate bug reports will be closed.
-A bug report simply stating that Darktable crashes is unhelpful, so please fill in most of the items below and provide detailed information.
--->
+**Introduction**
+_This section is provided for information, please delete, if understood_
 
-**Describe the bug**
-<!-- A clear and concise description of what the bug is. -->
+Before raising a bug/issue please check that it has not already been reported 
+by searching the existing https://github.com/darktable-org/darktable/issues on GitHub.
+
+Please be aware that most developers work on this project in their free time.
+The more information you can provide in the initial report the quicker we can help.
+Therefore please help to be as accurate and concise as possible!
+
+For further information, please see the https://github.com/darktable-org/darktable/wiki#reporting-bugs.
+---------------------------------------------------------------
+
+
+**Describe the bug/issue**
 
 
 **To Reproduce**
-<!-- Provide detailed steps that can reproduce the behavior, such as:
+_Please amend or remove, what is/isn't applicable_
 1. Go to '...'
 2. Click on '...'
 3. Scroll down to '...'
 4. See error
--->
-
 
 **Expected behavior**
-<!-- A clear and concise description of what you expected to happen. -->
 
 
 **Screenshots**
-<!-- If applicable, add screenshots to help explain your problem. -->
+_Please remove, if not applicable (e.g. no screenshots)_
+
+**Screencast**
+_Please remove, if not applicable (e.g. no screencast)_
+
+**Which commit intruduced the error**
+_If possible, please try using `git bisect`. Please remove, if you do not know, what bisect is, or if you do, but not applicable_
+
+_A bisect is much appreciated and can significantly simplify the developer's job._
+_HowTo: https://github.com/darktable-org/darktable/wiki#finding-bug-causes and https://www.youtube.com/watch?v=D7JJnLFOn4A_
+
+
 
 
-**Platform (please complete the following information):**
- - Darktable Version: <!-- [e.g. 2.6.0] -->
- - OS: <!-- [e.g. Windows 8.1, Gentoo Linux] -->
- - <!-- OpenCL activated or no? -->
- - <!-- Which graphics card and driver version -->
+**Platform**
+_Please fill as much as possible inccluding "unknown" and remove those lines, which are clearly not applicable_
```

Comment:
A typo in including word (2 'c' instead of 1) and missing comma maybe before including and after "unknown"?

Reply:
English is weird language that seems to not like as many commas as normal languages